tests/unit/aiplatform/test_end_to_end.py

```diff
@@ -370,7 +370,7 @@ def test_dataset_create_to_model_predict_with_pipeline_fail(
         mock_python_package_to_gcs.assert_called_once_with(
             gcs_staging_dir=test_training_jobs._TEST_BUCKET_NAME,
             project=test_training_jobs._TEST_PROJECT,
-            credentials=initializer.global_config.credentials,
+            credentials=None,
```

Comment:
Training Job passes self.credentials to this method: https://github.com/googleapis/python-aiplatform/blob/dev/google/cloud/aiplatform/training_jobs.py#L1597

So it seems like this should be the credentials and not None.